client/src/components/CardList/CardList.stories.js

```diff
@@ -0,0 +1,37 @@
+import _ from "lodash";
+import React from "react";
+
+import { CardList } from "./CardList";
+
+export default {
+  title: "CardList",
+  component: CardList,
+};
+
+const Template = (args) => <CardList {...args} />;
+
+const elements = [
+  {
+    display: "Display 1",
+    href: "",
+    children: [
+      { display: "Child 1", href: "" },
+      { display: "Child 2", href: "" },
+      { display: "Child 3", href: "" },
+    ],
+  },
+  {
+    display: "Display 2",
+    href: "",
```

Comment:
Maybe name this one "Title with href", add an `href` value (`#` should be fine, as a do-nothing but non-empty value)

Reply:
Done!